static/ckeditor/ckeditor/plugins/mjAccordion/ckeditor.html

```diff
@@ -0,0 +1,48 @@
+<textarea id="ckContent"></textarea>
+<div id="buttons_container">
+	<input type="button" value="Save Changes" id="saveChanges" onClick="save()" />
+	<input type="button" value="Cancel" id="cancel" onClick="window.close()" />
+</div>
+
+
+<style type="text/css">
+* {margin:0; padding:0;}
+input {outline:none; padding:6px 9px; margin:2px; cursor:pointer; font-family:Arial, Helvetica, sans-serif; -webkit-border-radius:3px; border-radius:3px; -webkit-box-shadow:inset 0 1px 0 rgba(255, 255, 255, 0.3), 0 1px 1px rgba(0, 0, 0, 0.15); box-shadow:inset 0 1px 0 rgba(255, 255, 255, 0.3), 0 1px 1px rgba(0, 0, 0, 0.15); background-color:#e0e0e0; color:#666; text-shadow:none; border:1px solid #ababab;}
+#saveChanges {background-color:#63a730; color:#fff; border:1px solid #62a62f; background-image:-webkit-linear-gradient(#7ebe4c,#63a730); background-image:linear-gradient(#7ebe4c,#63a730);}
+#cancel {background-image:-webkit-linear-gradient(#e8e5e5,#e0e0e0); background-image:linear-gradient(#e8e5e5,#e0e0e0);}
+input:active {-webkit-box-shadow:inset 0 0 6px rgba(0, 0, 0, 0.2); box-shadow:inset 0 0 6px rgba(0, 0, 0, 0.2)}
+#buttons_container {position:fixed; width:100%; height:35px; bottom:.5em; right:.5em; text-align:right; z-index:9999;}
+</style>
+
+<script type="text/javascript" src="//cdn.ckeditor.com/4.5.9/standard/ckeditor.js"></script>
+<script type="text/javascript">
+// configure your editor here and pass it to "editor" variable

```

Comment:
Could you get the config for this to match our main CKeditor config?

Mostly for the toolbar.